packages/components/src/ecosystem_bar/index.jsx

```diff
@@ -0,0 +1,43 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { useRef } from 'react';
+import { useOnceSwipeTracking, useScrolled } from '@/src/ecosystem_bar/hooks';
+import styles from './index.module.scss';
+
+const EcosystemBar = ({ items, onFirstSwipe, onItemClick }) => {
+  const navRef = useRef(null);
+  const scrolled = useScrolled(navRef);
+
+  useOnceSwipeTracking(navRef, onFirstSwipe);
+
+  return (
+    <ul className={clsx(styles.root, { [styles.scrolled]: scrolled })} ref={navRef}>
+      {items.map((item) => (
+        <li key={item.key} className={styles.item}>
+          <a
+            className={clsx(styles.link, { [styles.isActive]: item.isActive })}
+            href={item.link}
+            rel="noreferrer noopener"
+            target={item.isActive ? '_self' : '_blank'}
+            onClick={() => onItemClick(item.event)}
```

Comment:
By moving to render function, seems like will be a good idea to extract it into a handler function as well inside of a render function. 
This way the function is being recreated on every re-render, and it will slightly impact the performance, as I remember. Would appreciate second opinion from @inomn

Reply:
Let's use our 'bind-way', to be consistent: https://github.com/good-hood-gmbh/nebenan-frontend/blob/6c232124adf70866f856c04d7b31198b5b86470c/client/components/ecosystem_nav_bar/index.jsx#L30